addons/project/static/tests/tours/project_task_history.js

```diff
@@ -112,6 +112,41 @@ registry.category("web_tour.tours").add("project_task_history_tour", {
                 throw new Error(`Expect description to be ${expected}, got ${p}`);
             }
         }
+    }, {
+        content: "Open tab Tasks.",
+        trigger: 'button[data-menu-xmlid="project.menu_project_management"]',
+        async run() {
+            $('button[data-menu-xmlid="project.menu_project_management"]').click();
+            await new Promise((r) => setTimeout(r, 1000));
```

Comment:
Why do you need to use a setTimeout ?

Reply:
This come from this comment https://github.com/odoo/odoo/pull/171090#discussion_r1666698400, i think i didn't do it right those

Reply:
Generally, adding a timeout to a test is an error. Issue:
- Slows down the test suite (3sec isn't much but on thousands of runbots it's expensive)
- Adds indeterminism. The test could fail indeterminately.

Solution:
- Check that an interface element has changed to validate that your saveis done ?

Reply:
Yes i removed